.github/pull_request_template.md

```diff
@@ -18,3 +18,10 @@ Please tick as appropriate and edit the reasons (e.g.: "backport: not needed bec
 
 Please refer to the [contributing guidelines](https://github.com/Mbed-TLS/mbedtls/blob/development/CONTRIBUTING.md), especially the
 checklist for PR contributors.
+
+Help make review efficient:
+* Multiple simple commits
+  - please structure your PR into a series of small commits, each of which does one thing
+* No force-push
+  - please do not force-push to update your PR - just add new commit(s)
+  - use `git merge` (or the GitHub web interface) to resolve conflicts - not `git rebase`
```

Comment:
I think that our development process says the opposite, in case of conflict we prefer a rebase.

Reply:
I'd say that it depends: early in the review process, a merge can be annoying. Especially if that's during the first review, and the second reviewer has to review some coding, a merge and more coding. At the other extreme, when both reviewers have approved but there's a conflict, resolving through merging can be easier to handle sometimes. However `git range-diff` tends to make rebases easy to review. So I would tend to favor rebasing as the default strategy, without prohibiting merges.

Reply:
That's the advantage of a link to https://mbed-tls.readthedocs.io/en/latest/reviews/review-for-contributors/  - it has rooms for more details. I think here we should stick with the simple rule that the _normal_ way to rework a PR is by adding commits, and refer to the extended version for more details.

Reply:
OK, I'll remove that line and link to the longer document where we can present more details